### DISCUSSION TITLE
What is your current understanding on how to Get Published?

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#7] [General]
It's a common misconception that because you work on a project, that you will be offered First Author Publication, let alone publication on any level.  
Here are some questions I have for the group that I'm curious understanding for preparing material.

1. What are your basic understanding on how to get published?  OR if not you knowledge base is zero, say that. 
2. Did your current PI make it clear to you that your work, if successful, will warrant you to be an author on the publication?
3. Is being non-first author an acceptable goal, or are you only interested in first-author publication endeavors?

**Comment by spulido1** [comment]
1. Communicate with your mentor that one of your goals is getting published. Be open to helping out in any way possible which can look like: performing literature review, drafting and editing formal research proposal, meeting with a healthcare analytics team (for power analysis--should be able to explain objective of project during meeting), submitting to IRB, performing data collection, writing paper, etc. 
2. Yes. 
3. I am interested in getting published as many times as possible. First-author status is always a good goal. Second or third author status is great too--earning a publication is always something to be proud of.

**Comment by Jdlopez0108** [comment]
1. What are your basic understanding on how to get published? OR if not you knowledge base is zero, say that.
 I have pretty limited knowledge in this area. I would just think to be explicit about your interests while having some humility about the actual amount of work you contributed to the project. 

2. Did your current PI make it clear to you that your work, if successful, will warrant you to be an author on the publication?
No, I haven't had this conversation. There are 3 rounds of the clinical trial, with each round being 6 months long. There was a lot of work and planning before I showed up so I don't really expect to.

3. Is being non-first author an acceptable goal, or are you only interested in first-author publication endeavors?
Definitely. I would be happy to have my name anywhere. :-)

**Comment by BlakeAcebo** [comment]
1.  In truth, I have a limited understanding of what amount of contribution warrants an "authorship". I would assume that the PI and lead scientists would have final say. In my past experience (Non-clinical research), the lead scientist often had the most say, as the PI was mostly overseeing multiple experiments and teams, thus he wasn't familiar with the extent of work that each team member contributed to each project. 

2. No, my current PI and I have not had that conversation. Honestly, the actual experiment aspect is much more premature that I thought and we are still awaiting IRB approval. 

3. I do not care about first authorship particularly, just having my name on a project would be cool. Perhaps if I created the hypothesis and clinical question behind the study then I would fee different, but that is not the case for most research.

**Comment by bguill3** [comment]
1. I am actually not sure how to get published. As stated at the first meeting, prior to this program I did not have any experience with research. 
2. Yes. But we only very briefly discussed this. 
3. I think just having my name published on anything is a great goal to have, especially for someone like myself who is just getting into the realm of research. As I gain more exposure and experience, I would at some point like to be able to be first author.

**Comment by mizquierdo2** [comment]
1. I relate the application process to medical school as the same as how to get published to understand it on a limited level. I understand you have to apply to multiple journals to find which one you will be accepted to and I am not sure if you have to pay to apply or how long the review board on a journal takes.
2. Somewhat. We talked about my desire to be published and through conversation they have mentioned a couple times that "when published"... 
3. I am okay with any form of authorship as I know this project is ongoing.

**Comment by jaq-serrano** [comment]
1. My knowledge base is essentially 0. I have never gone through the process of trying to get a publication. 
2. No. She did say I could initiate a different project or highlight a perspective they haven't covered yet but I am not exactly sure how to go about that or if it could lead to a publication. 
3. Being a non-first author is an acceptable goal. I am also interested in continuing this research after this summer, so hopefully I gain a better perspective on how to get a first-author publication.
